src/components/data/EntitiesContent.js

```diff
@@ -146,7 +149,12 @@ const ToolDrawer = _.flow(
               text: 'Workflow'
             }),
             h(ModalToolButton, {
-              onClick: !openDataExplorerInSameTab ? onDismiss : undefined,
+              onClick: () => {
+                Ajax().Metrics.captureEvent(Events.workspaceDataOpenWithDataExplorer, {
+                  ...extractWorkspaceDetails(workspace.workspace)
+                })
+                if (!openDataExplorerInSameTab) onDismiss()
```

Comment:
nit: we typically use short circuiting for these changes
```suggestion
                !openDataExplorerInSameTab && onDismiss()
```

Reply:
Fixing!